src/actions/taskActions.ts

```diff
@@ -1,219 +1,263 @@
+  boardRef,
+  position,
+}: {
+  startColumn: dragDropColumn;
+  endColumn: dragDropColumn;
+  boardTag: string;
+  position: number;
+  boardRef: string;
 }) => {
-    return async (dispatch: Function) => {
-        try {
-            dispatch(
-                boardAction.moveTaskBetweenBoardTaskColumns({
-                    startColumn,
-                    endColumn,
-                })
-            );
+  return async (dispatch: Function) => {
+    try {
+      dispatch(
+        boardAction.moveTaskBetweenBoardTaskColumns({
+          startColumn,
+          endColumn,
+        })
+      );
 
-            await moveTask({
-                boardTag,
-                boardRef,
-                position,
-                status: endColumn.name,
-            });
-        } catch (err) {
-            dispatch(boardAction.setError(true));
-            const errorMsg = extractMessage(err);
-            toastError(extractMessage(errorMsg));
-            dispatch(boardAction.setErrorMsg(errorMsg));
-        }
-    };
+      await moveTask({
+        boardTag,
+        boardRef,
+        position,
+        status: endColumn.name,
+      });
+    } catch (err) {
+      dispatch(boardAction.setError(true));
+      const errorMsg = extractMessage(err);
+      toastError(extractMessage(errorMsg));
+      dispatch(boardAction.setErrorMsg(errorMsg));
+    }
+  };
 };
 
+// export const addImageToTaskCover = ({
+//   boardTag,
+//   boardRef,
+//   imageName,
+//   imageObj,
+//   imageUrl,
+// }: {
+//   boardTag: string;
+//   boardRef: string;
+//   imageName: string;
+//   imageObj: any;
+//   imageUrl: string | undefined;
+// }) => {
+//   return async (dispatch: Function) => {
+//     try {
+//       const response = await addTaskCoverImage({
+//         boardTag,
+//         boardRef,
+//         imageName,
+//         imageObj,
+//         imageUrl,
+//       });
+
+//       console.log(response);
+//     } catch (err) {
+//       console.log(err);
+//     }
+//   };
+// };
+
 export const createComment = ({
-                                  boardRef,
-                                  message,
-                                  mentionedUsers,
-                                  taskId,
-                                  user,
-                                  columnId,
-                              }: {
-    boardRef: string;
-    user: string;
-    taskId: number;
-    message: string;
-    columnId: number;
-    mentionedUsers: string[];
+  boardRef,
+  message,
+  mentionedUsers,
+  taskId,
+  user,
+  columnId,
+}: {
+  boardRef: string;
+  user: string;
+  taskId: number;
+  message: string;
+  columnId: number;
+  mentionedUsers: string[];
 }) => {
-    return async (dispatch: Function) => {
-        try {
-            console.log({
-                boardRef,
-                message,
-                mentionedUsers,
-                taskId,
-                user,
-            });
-            // const response = await createCommentReq({
```

Comment:
Please ensure that any unused code is removed instead of being commented out. Can you please explain why this specific code was commented out?